src/lib.rs

```diff
@@ -132,7 +132,8 @@ impl DecodeReaderBytesBuilder {
         }
     }
 
-    /// Build a new decoder that wraps the given reader.
+    /// Build a new decoder that wraps the given reader. The default buffer
+    /// size is 8kb.
```

Comment:
The default buffer size is not an API guarantee and shouldn't be in the docs like this.

Reply:
I agree that it's not a guarantee but it's still worth mentioning even if we explicitly say it's not an API guarantee.  I can copy the language from `std::io::BufReader` if you like https://doc.rust-lang.org/std/io/struct.BufReader.html#method.new

Reply:
Obviously it's your call, but considering the constant in question is 2 lines away and the consistency with the standard library, I'm not sure I understand the big deal.

Reply:
But if you're strongly against it, that's fine.

Reply:
Please remove this change.